R/Layer.R

```diff
@@ -0,0 +1,134 @@
+#' #' Create a custom Layer
+#' 
+#' @param classname the name of the custom Layer.
+#' @param initialize a function. This is where you define the arguments used to further
+#'  build yolur layer. For example a dense layer, would take the `units` argument.
+#'  You should always call \code{super()$`__init__()`} to intialized the base 
+#'  inherited layer.
+#' @param build a function that takes `input_shape` as argument. This is where you will 
```

Comment:
The Keras [documentation](https://keras.io/layers/writing-your-own-keras-layers/) says that the `build` method should set the attribute `self$built <- TRUE` at the end, which can be done by calling the parent `super()$build()`. I'm not completely sure what the `built` attribute is used for, but [this](https://www.tensorflow.org/api_docs/python/tf/keras/layers/Layer?version=stable) page implies that it's needed for when the user wants to call `build()` manually. 
Do you have any additional insight here?